app/views/CharitiesCommunityAmateurSportsView.scala.html

```diff
@@ -37,7 +37,7 @@ <h1 class="header-margin-top">@messages("charities.community.amateur.sports.titl
     </div>
 
     @if(!appConfig.performanceTest){
-        <div id="charities_community_amateur_clubs"></div>
+        <div id="HMRC_FIXED_1"></div>
```

Comment:
mainTemplate already can a container Div for nuance with an id of "HMRC_Anchored_1". However the new div above with id "HMRC_FIXED_1" will still work if that's what nuance is searching for

Reply:
The <div id="HMRC_Anchored_1"></div> is for all pages so webchat can exist when navigating different pages. This needs to be on every page in the service. The
<div id="HMRC_FIXED_1"></div>
is so Nuance can actually offer webchat in the specific <div>on a page. I think this will be rolled out to the existing 10 pages at some point so maybe worth refactoring at some point. (